features/support/env.rb

```diff
@@ -1,5 +1,9 @@
 require 'aruba/cucumber'
 
+require_relative '../../spec/helpers/faker'
+
+ModuleSync::Faker.working_directory = File.expand_path("#{Aruba.config.working_directory}/faker")
```

Comment:
This should also work
```suggestion
ModuleSync::Faker.working_directory = File.expand_path('faker', Aruba.config.working_directory)
```

Reply:
OK